src/components/Menu.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import Basket from './Basket';
+
+
+class Menu extends React.Component {
+  constructor() {
+    super();
+    this.state = {basket:{}}
```

Comment:
Basket state appears duplicated across Menu and Basket components. It would be better to store it on place such as `App` and share it using props